frontend/src/content/news.json

```diff
@@ -1,5 +1,10 @@
 {
   "2021": [
+    {
+      "date": "2021-12-06",
+      "title": "Metabolic Atlas 2.2 fixes bugs",
+      "text": "<b>Metabolic Atlas 2.2</b> is out. In this minor release we focused on resolving several serious bugs that were discovered since the previous release, primarily around the Interaction Partner and Map Viewer tools, and also some smaller bugs that were previously reported. More details can be found <a href='https://github.com/MetabolicAtlas/MetabolicAtlas/releases/tag/2.2' target='_blank'>on GitHub</a> or <a href='https://zenodo.org/badge/latestdoi/53664497' target='_blank'>on Zenodo</a>."
+    },
```

Comment:
I can open find one release, i.e. `Metabolic Atlas 2.1` on the Zenodo page https://zenodo.org/badge/latestdoi/53664497 . Should each release has its own ID?

Reply:
There is no doi for the latest release since it hasn't been made yet (it will be done automatically after the GH release is made, which is why the "latestdoi" is used), but it would make sense to go back at the previous releases and link directly to the right doi from Zenhub, since those exist
. Is this what you have in mind?

Reply:
I hope the above is what you meant, so aimed to address it.